src/components/List/List.tsx

```diff
@@ -126,7 +126,7 @@ export const List = React.forwardRef<HTMLUListElement, Props>(
         };
 
         return isControlledNavigation ? (
-            <ul {...rest} ref={ref} {...block({ isDivided, ...rest })}>
+            <ul {...rest} ref={ref} {...block({ isDivided, ...rest })} role="listbox">
```

Comment:
:+1:

Reply:
You were right. Role `list` is for static lists, while `listbox` is for selectable lists, which is what this component is about. Since we have `onKeyDown` and `tabindex` we should use `listbox`.